### DISCUSSION TITLE
problems moving file

### DISCUSSION THREAD

**Discussion by Cryptodarktrader** [SilentVoid13/Templater#1217] [Help] [answered]
Hello, I am creating a template for my personal contacts, more or less I already have everything, but the only problem that I have not been able to solve is the code so that when I add the template to a blank note, that note will automatically move if it is not already there. the folder where I save my contacts.

this doesn't seem to work
<% wait tp.file.move("/Example folder/" + tp.file.title) %>

**Comment by Zachatoo** [comment, marked as answer]
I believe it should be

```js
<% await tp.file.move("/Example folder/" + tp.file.title) %>
```

If that doesn't work, it may be something else in the template causing the issue. Having the error in the console would be helpful in diagnosing the issue.

**Comment by Cryptodarktrader** [comment]
It worked perfectly, I saw that the problem is my bad English, which was putting wait instead of await.

I have tried it and it moves the file perfectly whether I place the title first and load the template or if I load it before adding the title.


I think the mistake was that I tried to put it before between notes

%----
----%

and apparently it doesn't take the code well like that
